content/src/content/jcr_root/apps/core/wcm/components/modal/v1/modal/clientlibs/site/css/modal.css

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright 2019 Adobe Systems Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+.tingle-modal-box__content {
+    max-height: 80vh;
+    overflow: auto;
+    margin-top: 5vh;
+    margin-bottom: 5vh;
+}
+
+.tingle-modal-box {
+    background: #fff;
```

Comment:
Is this necessary? Looks like it's already the default https://github.com/adobe/aem-core-wcm-components/pull/539/files#diff-b027cee413c80d2945576e1d27927befR96